### PR TITLE
DM-13914 First draft of technote

### PR DESCRIPTION
**Pull request by mtpatter**
First draft of technote

### CODE REVIEW

File: index.rst
Comment (ebellm) [inline index.rst]: perhaps link to DMTN-028

File: index.rst
Comment (ebellm) [inline index.rst]: Maybe say "science users can then subscribe"?  Unless you mean "consumers" in the Kafka context (which would also be fine)

File: index.rst
Comment (ebellm) [inline index.rst]: (e.g., Kubernetes)

File: index.rst
Comment (ebellm) [inline index.rst]: note that `docker-compose up -d` creates a network `alert_stream_lsst_default` (added `_lsst`)--I would harmonize this throughout the document.

File: index.rst
Comment (ebellm) [inline index.rst]: when I run this I get an error:

```docker run -it --rm --network=alert_stream_lsst_default alert_stream python bin/filterStream.py full-stream 1 20
usage: filterStream.py [-h] [--group GROUP] [--stampDir STAMPDIR]
                       [--decode | --decode-off]
                       topic
filterStream.py: error: unrecognized arguments: 1 20
```

File: index.rst
Comment (ebellm) [inline index.rst]: This also gives me an error:

```
(base) D-10-155-91-135:data ebellm$ docker run -it --rm \
>            --name=sender \
>            -v $PWD:/home/alert_stream/data:ro \
>            --network=alert_stream_lsst_default \
>            alert_stream python bin/sendAlertStream.py full-stream
usage: sendAlertStream.py [-h] [--encode | --encode-off] topic avrofile
sendAlertStream.py: error: the following arguments are required: avrofile
```

**Review by ebellm — CHANGES_REQUESTED** [review]
I'm having trouble reproducing the single-container case--thoughts?

File: index.rst
Comment (mtpatter) [reply index.rst]: Sorry, this (and everything in the technote) will only work after DM-13476 is merged.  It should work if you check out that branch.

File: index.rst
Comment (mtpatter) [reply index.rst]: Again here need DM-13476.

File: index.rst
Comment (mtpatter) [reply index.rst]: Do you mean to say that you want me to change the network name or that you observed that _lsst is added to the network name?  I used that name because the network name should be autocreated from the directory name, which is alert_stream.  Is that not true for you?

Comment (ebellm) [reply index.rst]: Ah, I see--problem on my side, then (I have another `alert_stream` directory)...

File: index.rst
Comment (ebellm) [reply index.rst]: Okay, one more PEBKAC problem: when I run this from the `alert_stream/data` directory it exits silently, sending no alerts.  I'm guessing it's failing to find the packets but I don't know why.

Comment (mtpatter) [reply index.rst]: My keyboard/chair.  I had wanted to change where the data directory is mounted but never committed it.  Changed and pushed to tickets/DM-13476 so it should work now.  Sorry about that.

**Review by ebellm — APPROVED** [review]
Looks good to go after the small text changes.

**Pull request closed without merging**
